browser-test/src/admin_program_visibility.test.ts

```diff
@@ -125,7 +126,7 @@ describe('Validate program visibility is correct for applicants and TIs', () =>
     await adminTiGroups.gotoAdminTIPage()
     await adminTiGroups.editGroup('groupTwo')
     // GroupTwo will have the TestUser as the TI, so it will be visible to this TI only
```

Comment:
Can we be a little more specific in this comment so future readers understand why we needed to get this from the config? How about "Add the TEST_USER_DISPLAY_NAME user to GroupTwo, which is the user that gets logged into upon calling loginAsTestUser()" ? Here and in the line below as well.

Reply:
Done